steps/Proposals.tsx

```diff
@@ -0,0 +1,609 @@
+import cls from 'classnames';
+import utilsStyles from '@styles/utils.module.scss';
+import Header from '@components/Header/Header';
+import styles from '../components/Footer/Footer.module.scss';
+import styles1 from '@styles/stepsPages.module.scss';
+import ButtonRound, { BUTTON_ROUND_COLOR, BUTTON_ROUND_SIZE } from '@components/ButtonRound/ButtonRound';
+import ColoredIcon, { ICON_COLOR } from '@components/ColoredIcon/ColoredIcon';
+import ArrowRight from '@icons/arrow_right.svg';
+import ArrowLeft from '@icons/arrow_left.svg';
+import Correct from '@icons/correct.svg';
+import Wallet from '@icons/wallet.svg';
+import Touch from '@icons/touch.svg';
+import Dots from '@icons/vertical_dots.svg';
+import { backRoute, replaceRoute } from '@utils/router';
+import { Proposal } from '@ixo/impactxclient-sdk/types/codegen/cosmos/gov/v1beta1/gov';
+import { useRouter } from 'next/router';
+import Anchor from '@components/Anchor/Anchor';
+import Loader from '@components/Loader/Loader';
+import SadFace from '@icons/sad_face.svg';
+import Thumbsup from '@icons/thumbs-up.svg';
+import Thumbsdown from '@icons/thumbs-down.svg';
+import NoWithVeto from '@icons/no-with-veto.svg';
+import Abstain from '@icons/abstain.svg';
+import Success from '@icons/success.svg';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper.min.css';
+import { QueryProposalsRequest } from '@ixo/impactxclient-sdk/types/codegen/cosmos/gov/v1beta1/query';
+import useQueryClient from '@hooks/useQueryClient';
+import { broadCastMessages } from '@utils/wallets';
+import { StepConfigType, StepDataType, STEPS } from 'types/steps';
+import { defaultTrxFeeOption } from '@utils/transactions';
+import { cosmos } from '@ixo/impactxclient-sdk';
+import { TRX_MSG } from 'types/transactions';
+import { KEPLR_CHAIN_INFO_TYPE } from 'types/chain';
+import { WalletContext } from '@contexts/wallet';
+import { ChainContext } from '@contexts/chain';
+import IconText from '@components/IconText/IconText';
+import { ViewOnExplorerButton } from '@components/Button/Button';
+
+type RequestProposalsProps = {
+    onSuccess: (data: StepDataType<STEPS.select_and_review_proposal>) => void;
+    onBack?: () => void;
+    data?: StepDataType<STEPS.select_and_review_proposal>;
+    config?: StepConfigType<STEPS.select_and_review_proposal>;
+    header?: string;
+};
+
+type FooterProps = {
+    onBackUrl?: string;
+    onBack?: (() => void) | null;
+    backLabel?: string;
+    onCorrect?: (() => void) | null;
+    correctLabel?: string;
+    onForward?: (() => void) | null;
+    forwardLabel?: string;
+    showAccountButton?: boolean;
+    showActionsButton?: boolean;
+    selectVoteAction?: (() => void) | null;
+};
+
+interface VoteBtnProps {
+    backgroundColor: string;
+    children: React.ReactNode;
+    onClick: () => void
+}
+
+const VoteButton = ({ backgroundColor, children, onClick }: VoteBtnProps) => {
+    const VoteBtnsStyle = {
```

Comment:
@RafeeqWeideman-YDS This `VoteButton` component can be on it's own - cleanup the code in this file and only keep related code